trip-ninja-docs/src/navbar/NavBar.css

```diff
@@ -1,11 +1,81 @@
 
-.navbar {
+.navbar-top {
+  background-color: var(--grey-lighter);
+  height: 90px;
+  width: 100%;
+  overflow-x: auto;
+}
+
+.navbar-bottom {
   background-color: var(--grey-lighter);
   border-bottom: 1px solid var(--grey-mid);
+  height: 60px;
+  overflow-x: auto;
+  overflow-y: hidden;
 }
 
 .logo-text {
   color: var(--primary);
   font-weight: 800;
   font-size: 22px; 
+}
+
+.nav-link {
+  margin-right: 10px;
+  font-weight: 500;
+  display: inline-block;
+  position: relative;
+  white-space: nowrap;
+  color: var(--grey-dark);
+}
+
+.nav-link:hover, #selected-link {
+  color: var(--primary);
+  text-shadow: 0px 0px 1px var(--primary), 0px 0px 1px var(--primary);
```

Comment:
I don't think this is in the design spec - it gives it a weird fuzzy appearance!

Also - should be `font-weight: 700`

Reply:
sounds good! updated!